extensions/guacamole-auth-duo/src/main/resources/translations/it.json

```diff
@@ -0,0 +1,13 @@
+{
+
+    "DATA_SOURCE_DUO" : {
+        "NAME" : "Duo TFA Backend"
```

Comment:
If this is the Italian translation of the English "Duo TFA Backend", then this can stay - otherwise, no need to include it.